# MmwObject
uint8 id                 # 目标id
float32 dist_long        # 目标纵向距离
float32 dist_lat         # 目标横向距离
float32 vrel_long        # 目标纵向速度
float32 vrel_lat         # 目标横向速度
uint8 sector_number      # 扇区编号